strawberry/resolvers.py

```diff
@@ -114,3 +119,18 @@ async def await_result(result):
 
     _resolver._is_default = not field.base_resolver  # type: ignore
     return _resolver
+
+
+def strawberry_info_from_graphql(
+    field: FieldDefinition, info: GraphQLResolveInfo
+) -> Info:
+    return Info(
+        field_name=info.field_name,
+        context=info.context,
+        root_value=info.root_value,
+        variable_values=info.variable_values,
+        return_type=field.type,
+        operation=info.operation,
+        path=info.path,
+        original_info=info,
+    )
```

Comment:
Hmm I wonder if this should somehow be tied into our new [schema_converter](https://github.com/strawberry-graphql/strawberry/blob/update-resolver-info-type/strawberry/schema/schema_converter.py) system. The only complexion is that it's _from_ a `graphql-core` type, and not _to_.

Reply:
Ah that looks interesting

Reply:
Our goal was to create an abstraction layer between the `graphql-core` code and the Strawberry backend

Reply:
@BryceBeagle can you think of a good place for that? 

we can also do this for now and then improve it later 😊

Reply:
I was thinking about this, but I don't have any great answers yet. It may be good to have both `to` and `from` converters, but that's a lot of effort :sweat_smile:. How do we currently convert from graphql-core objects to Strawberry objects elsewhere in the codebase?

Reply:
this is the only place where we convert graphql-core objects :)

Reply:
Hmm, that certainly reduces motivation to create a whole abstracted system :thinking:

Reply:
yeah, I think we can figure out this later 😊 maybe we can have something in StrawberryResolver for this